src/client/components/threatbar/threatbar.js

```diff
@@ -243,48 +246,47 @@ class Threatbar extends React.Component {
             <ModalBody>
               <FormGroup>
                 <Label for="title">Title</Label>
-                <Input type="text" name="title" id="title" disabled={ this.props.G.threat.owner !== this.props.playerID } autoComplete="off" value={this.state.title} onBlur={(e) => this.props.moves.updateThreat("title", e.target.value)} onChange={(e) => this.updateState("title", e.target.value)} />
+                <Input type="text" name="title" id="title" disabled={this.props.G.threat.owner !== this.props.playerID} autoComplete="off" value={this.state.title} onBlur={(e) => this.props.moves.updateThreat("title", e.target.value)} onChange={(e) => this.updateState("title", e.target.value)} />
               </FormGroup>
               <FormGroup>
                 <Label for="type">Threat type</Label>
-                <Input type="select" name="type" id="type" disabled={ this.props.G.threat.owner !== this.props.playerID } value={this.props.G.threat.type} onChange={(e) => this.props.moves.updateThreat("type", e.target.value)}>
-                  <option value="S">Spoofing</option>
-                  <option value="T">Tampering</option>
-                  <option value="R">Repudiation</option>
-                  <option value="I">Information Disclosure</option>
-                  <option value="D">Denial of Service</option>
-                  <option value="E">Elevation of Privilege</option>
+                <Input type="select" name="type" id="type" disabled={this.props.G.threat.owner !== this.props.playerID} value={this.props.G.threat.type} onChange={(e) => this.props.moves.updateThreat("type", e.target.value)}>
+                  {
+                    Object.keys(STARTING_CARD_MAP).map(suit => (
+                      <option value={suit}>{getTypeString(suit, this.props.gameMode)}</option>
```

Comment:
In react, when a component is added through a loop, one should add a `key`, see [the docs](https://reactjs.org/docs/lists-and-keys.html#keys).

So, you should add a property like 
```js
key={`threat-category-${suit}`}
```